playground/PrivateDNS.bicep

```diff
@@ -0,0 +1,21 @@
+param zoneName string
+param vnetLinks array
+
+//create the dnszone resource
+resource privateDnsZone 'Microsoft.Network/privateDnsZones@2018-09-01' = {
+  name: zoneName
+  location: 'global'
+}
+
```

Comment:
Try to add `@batchSize(1)` here to make sure that the VNet links are deploy in serial mode and not both at the same time.

Reply:
Thanks, will try that.  Where do you find the documentation for those parameter guidelines, I see an example of this in the BICEP doc where they limit characters for things like a storage account name.  I'm wondering what else you can do, given the example you've listed above.

Reply:
I'm getting an error trying to use `@batchsize(1)` - `cannot be used as a param decorator`

Reply:
@batchsize(1) is a decorator that should be added to a resource declaration using loops. You can find more information about it here: https://github.com/Azure/bicep/blob/main/docs/spec/loops.md#batch-size-decorator

Decorators that can be used for parameters can be found here.